stable/gcloud-sqlproxy/values.yaml

```diff
@@ -0,0 +1,43 @@
+## Google Cloud SQl Proxy image
+## ref: https://cloud.google.com/sql/docs/mysql/sql-proxy
+## ref: https://cloud.google.com/sql/docs/postgres/sql-proxy
+image: b.gcr.io/cloudsql-docker/gce-proxy:1.08
+
+## Specify a imagePullPolicy
+## 'Always' if imageTag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+imagePullPolicy: IfNotPresent
+
+## Replicas Set count
+replicasCount: 1
+
+## GCP service account key which has access to Cloud SQL instances
+## the key must be encoded with base64
+## e.g. `cat service-account.json | base64`
+gcpServiceAccountKey: 'Paste base64 encoded data here.'
```

Comment:
Would also help CI get past the following issue

<details>
Error: release gcloud--1252 failed: unable to decode "": [pos 79]: json: error decoding base64 binary 'Paste base64 encoded data here.': illegal base64 data at input byte 5
</details>